common/common.go

```diff
@@ -380,6 +380,16 @@ func writeTmpDockerfile(helper langs.LangHelper, dir string, ff *FuncFile) (stri
 		dfLines = append(dfLines, fmt.Sprintf("FROM %s", ri))
 		dfLines = append(dfLines, "WORKDIR /function")
 		dfLines = append(dfLines, helper.DockerfileCopyCmds()...)
+		
+		resources := ff.Copy_resources
+
+		for _, resource := range resources {
+			if Exists(resource.Resource.From){
```

Comment:
When checking if the file exists, you need to differentiate between absolute and relative paths. For absolute paths (those with a leading `/`), the above will work. For relative paths, you will need to resolve the full path prior to calling `Exists`. See for example, https://github.com/fnproject/cli/blob/17c045e83ccfc0ab8c39902c67f9858de8097ffb/common/common.go#L154